src/frontend/Komponenter/Behandling/Brev/BrevInnhold.tsx

```diff
@@ -204,14 +206,19 @@ const BrevInnhold: React.FC<Props> = ({
                             </ToKolonneLayout>
                         );
                     })}
-                    {brevSkalKunneRedigeres && (
-                        <LeggTilKnappWrapper>
+                    <LeggTilKnappWrapper>
+                        {brevSkalKunneRedigeres && (
                             <LeggTilKnapp
                                 onClick={leggAvsnittBakSisteSynligeAvsnitt}
                                 knappetekst="Legg til avsnitt"
                             />
-                        </LeggTilKnappWrapper>
-                    )}
+                        )}
+                        {settVisNullstillBrevModal && (
+                            <KnappMedLuftUnder onClick={() => settVisNullstillBrevModal(true)}>
```

Comment:
Hadde egentlige vært fint hvis vi kunde byttet ut gamle knapper til nye i en komponent når vi gjorde disse endringene
Vi gikk for å bruke nye i https://github.com/navikt/familie-ef-sak-frontend/pull/1637 og blir då en blandning då det er andre komponenter på den siden som bruker de gamle

Reply:
Ja, men usikker på om @eirarset er i gang med det i en egen branch? Siden denne knappen skal være "likestilt" med den over så må jeg isåfall bytte begge - og da begynner ballen rulle - for den "legg til knapp" brukes på 3-4 steder ...